src/components/Footer.jsx

```diff
@@ -0,0 +1,68 @@
+import { IoLocationSharp } from 'react-icons/io5';
+import { FaClock } from 'react-icons/fa';
+import { MdEmail } from 'react-icons/md';
+import { FaPhone } from 'react-icons/fa';
+import { FaInstagram } from 'react-icons/fa';
+import { FaFacebook } from 'react-icons/fa';
+import { FaYoutube } from 'react-icons/fa';
+
+import '../Footer.css';
+
+const Footer = () => {
+  return (
+    <div>
+      <div className="footer-main">
+        <div className="left-footer-content">
+          <div>
+            <h3>STUDENTSYNC MANAGEMENT APP</h3>
+            <p className="paragraph1">Student management platform</p>
+          </div>
+          <h4>Our Address</h4>
+          <div className="address">
+            <IoLocationSharp size={20} />
+
+            <p className="text-sm">
+              Gasabo District, Kinyinya Sector Murama Cell, Rusenyi Village KG 29 Ave 16, Kigali,
+              Rwanda
+            </p>
+          </div>
+          <div className="address">
+            <FaClock size={14} />
+            <p> Working Hours: Monday - Friday 08:00 - 17:00</p>
+          </div>
+        </div>
+
+        <div className="middle-footer-content">
+          <h4>Contact us</h4>
+          <p className="address">
+            <MdEmail /> info@thegymcapstone.rw
+          </p>
+          <p className="address">
+            <FaPhone className="phone-icon" /> 0788-322-223
+          </p>
+        </div>
+
+        <div className="right-footer-content">
+          <h4>Social media</h4>
+          <ul>
+            <li>
+              <FaInstagram />
+            </li>
+            <li>
+              <FaFacebook />
+            </li>
+            <li>
+              <FaYoutube />
+            </li>
```

Comment:
Please note that in the figma design the icons used where outlined not solit as you have used here. I would suggest that you take it on such as this using MUI icons as suggested before:

```suggestion
import InstagramIcon from '@mui/icons-material/Instagram';
import FacebookIcon from '@mui/icons-material/Facebook';
import YouTubeIcon from '@mui/icons-material/YouTube';

<li><InstagramIcon /></li>
<li><FacebookIcon /></li>
<li><YouTubeIcon /></li>

```

This matches the pre-installed package, and allows you to match the design.

**Please remember to the same for all the other icons in this component.**